.gitignore

```diff
@@ -129,4 +129,10 @@ dmypy.json
 .pyre/
 
 # JUnit file
-junit-test.xml
\ No newline at end of file
+junit-test.xml
+
+# VSCode
+.vscode/
```

Comment:
non-review comment: Global gitignore files are where I store all my editor files https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files

Reply:
Not a bad idea. Thanks!